index.bs

```diff
@@ -6827,7 +6832,7 @@ to protect the [=attestation object=] from [=man-in-the-middle attacks=].
 
 Under the assumption that a [=registration ceremony=] is completed securely, and that the [=authenticator=] maintains
 confidentiality of the [=credential private key=], subsequent [=authentication ceremonies=] using that [=public key
-credential=] are resistant to [=man-in-the-middle attacks=].
+credential=] are resistant to tampering from [=man-in-the-middle attacks=].
```

Comment:
I'm not sure "from" is the right preposition to use here. Perhaps it should be "by"?

Reply:
by or via are probably better

Reply:
I agree, thanks!